### PR TITLE
Fixed #67: Added profile links

### LINKED ISSUE #67: PROFILE LINKS MISSING

### PR DESCRIPTION
**Pull request by samyakbvs**
#67

### CODE REVIEW

**Comment by samyakbvs** [comment]
PLS MERGE

File: index.html [hunk 1]
```diff
@@ -843,6 +845,7 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/repeatedly"><i class="icon social_twitter"></i></a>
 											<a href="http://jp.linkedin.com/in/repeatedly" target="_self"><i class="icon social_linkedin"></i></a>
+											<a href="https://m.facebook.com/people/Masahiro-Nakagawa/1251439284" target="_self"><i class="icon social_facebook"></i></a>
```
Comment (0x48piraj) [inline index.html]: Remove the mobile facebook url

File: index.html [hunk 2]
```diff
@@ -1312,6 +1323,7 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/akshay_ratan"><i class="icon social_twitter"></i></a>
 											<a href="http://in.linkedin.com/pub/akshay-ratan/60/aa0/396" target="_self"><i class="icon social_linkedin"></i></a>
+											<a href="https://www.facebook.com/akshay.ratan" target="_self"><i class="icon social_facebook"></i></a>
```
Comment (0x48piraj) [inline index.html]: Keep the flow, the same.
Follow existing link structure.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
See my code reviews :+1:

File: index.html (hunk 1 above)
Comment (samyakbvs) [reply index.html]: What do you mean?

Comment (akshatnitd) [reply index.html]: @samyakbvs Provide the url for facebook website, rather than facebook mobile site

File: index.html (hunk 2 above)
Comment (akshatnitd) [reply index.html]: You can add target attribute before href. Even though, it much matters much, but adds to code quality!

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
Fixed issues @akshatnitd

**Review by akshatnitd — COMMENTED** [review]
Squash the commits and it is ready for merge :+1:

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@samyakbvs  Please provide a preview link else looks good 👍

**Comment by harshit98** [comment]
@samyakbvs 

- Provide preview link.
- Change PR name `Fixes #(issue-num): details of fix`.
- Avoid capitals. For e.g: 'PLS MERGE'. You can google about it.

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/s-a-m-2015.fossasia.org/

**Comment by samyakbvs** [comment]
Changes done @harshit98 and @ParthS007

**Review by the-codingninja — APPROVED** [review]

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**